frontend/src/components/share/modals/showCreateReverseShareModal.tsx

```diff
@@ -50,6 +53,12 @@ const Body = ({
       sendEmailNotification: false,
       expiration_num: 1,
       expiration_unit: "-days",
+
+      easyMode: false,
```

Comment:
See my comment about `easyMode`. If we remove `easyMode` the file would need to have a refactor.

Reply:
Yes you're right, but is it necessary ? Your choice